style.css

```
@@ -32,6 +32,24 @@
     from { opacity: 0; }
     to   { opacity: 1; }
 }
+:root{
+
+	--primary-color: #fff;
+	--secondary-color : #403e3e;
+	--extra-color:#efefef;
+
+
+
+
+}
+.dark-theme{
+	--primary-color: #171010;
+	--secondary-color : #fcf4f4;
+	--extra-color:#2B2B2B;
+}
+.bg{
+	background-color: var(--primary-color);
+}

```

Add background transition 
```suggestion
transition: background 500ms;
}

```

---

Okay, I am making the necessary changes and keeping it in the navbar